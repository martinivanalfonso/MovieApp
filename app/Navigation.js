import React from 'react'
import { createStackNavigator } from '@react-navigation/stack'
import Home from './screens/Home/Home'
import Movie from './screens/Movie/Movie'

const Stack = createStackNavigator()

export const Navigation = () => (
  <Stack.Navigator>
    <Stack.Screen name="Home" component={Home} options={{ headerShown: false }} />
    <Stack.Screen name="Movie" component={Movie} options={{ headerShown: false }} />
  </Stack.Navigator>
)

export default Navigation