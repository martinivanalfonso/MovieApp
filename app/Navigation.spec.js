import React from 'react'
import { shallow } from 'enzyme'
import Navigation from './Navigation'

describe('Navigation Component', () => {
  test('renders correctly', () => {
    const component = shallow(<Navigation />)
    expect(component).toMatchSnapshot()
  })
})
