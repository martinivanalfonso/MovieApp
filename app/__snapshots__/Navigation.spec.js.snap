// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component renders correctly 1`] = `
<StackNavigator>
  <Screen
    component={[Function]}
    name="Home"
    options={
      Object {
        "headerShown": false,
      }
    }
  />
  <Screen
    component={[Function]}
    name="Movie"
    options={
      Object {
        "headerShown": false,
      }
    }
  />
</StackNavigator>
`;
